tests/test_griddeddata.py

```diff
@@ -98,8 +100,14 @@ def test_GriddedData_longitude(data_tm5):
     assert str(data_tm5.longitude.units) == "degrees"
 
     lons = data_tm5.longitude.points
-    assert_allclose(lons.min(), -181.5, rtol=TEST_RTOL)
-    assert_allclose(lons.max(), 175.5, rtol=TEST_RTOL)
+    # iris >= 3.2 corrected an error in iris.cube.Cube.intersection
+    # see https://github.com/metno/pyaerocom/issues/588
+    iris_version = Version(metadata.version("scitools-iris"))
+    if iris_version >= Version("3.2"):
+        lon_min, lon_max = -178.5, 178.5  # correct values
+    else:
+        lon_min, lon_max = -181.5, 175.5
+    assert (lons.min(), lons.max()) == (lon_min, lon_max)
```

Comment:
why not to decorate the test with need_iris_32 and test only against the correct values?

``` python
@data_unavail
@need_iris_32
def test_GriddedData_longitude(data_tm5):
    """Test if longitudes are defined right"""
    assert str(data_tm5.longitude.units) == "degrees"

    lons = data_tm5.longitude.points
    assert (lons.min(), lons.max()) == (-178.5, 178.5)
```

Reply:
done